refinery/core/models.py

```diff
@@ -1067,7 +1066,7 @@ def __str__(self):
 
     def get_expanded_workflow_graph(self):
         return tool_manager.utils.create_expanded_workflow_graph(
-            ast.literal_eval(self.workflow_copy)
+            json.loads(self.workflow_copy)
```

Comment:
How will this work with existing analyses? The value stored in `workflow_copy` was a Python dict which is not going to be processed correctly by `json.loads()`.

Reply:
yes, this fell by the wayside through testing but i remember this now.  do you think we should check type here or remove the json.dumps() i added around self.workflow_copy? i think the later.

Reply:
I would guess a bunch of other code is relying on the existing serialization format, so it'd be safer to avoid making this change in this PR.